scripts/lib/common.sh

```diff
@@ -81,5 +83,8 @@ debug_msg() {
     if [ -n "$__indent_level" ]; then
         __indent="$( for each in $( seq 0 $__indent_level ); do printf " "; done )"
     fi
+    if [[ "${DEBUG}" = "true" ]]; then
+        set -x
+    fi
```

Comment:
Hi @tbutler4! Actually, I thought you should add this block *outside* the `debug_msg()` Bash function at the top of the file. Since this file is `source`'d from almost every other script right at the beginning of the script, having this block at the top of the `scripts/lib/common.sh` file outside of a function scope will essentially set the -x option for the remainder of the calling/sourcing script's execution.

Reply:
ahhh I see how that is more beneficial. I moved the block of code to the correct spot. This portion of #504 should be completed now I believe.